.github/workflows/vale.yml

```
@@ -0,0 +1,20 @@
+name: reviewdog
+on:
+  pull_request:
+  push:
+
```

```suggestion
on:
  pull_request:
  push:
         # trying and staging branches are for BORS config
        branches:
          - main
```

It appears it runs the tests both times since my change. I looked into the other repo's and that is how we do it :)